wp-parsely.css

```
@@ -34,7 +34,7 @@
 
 .parsely-recommended-widget-title {
 	font-size: 1.1em;
-	font-weight: bold;
+	font-weight: 700; /* Bold */
```

The comment is not needed.

```suggestion
	font-weight: 700;
```